portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,4 +44,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // get comment from form and add it to ArrayList
```

Comment:
nit: You don't need this comment. In general comments should explain "why", not "what". The code is clear enough to show the "what" by itself. Same below

Reply:
Appreciate the feedback, that's very helpful to know! I've removed those comments and pushed a fix.